pyqtgraph/graphicsItems/GradientLegend.py

```diff
@@ -15,17 +15,25 @@ def __init__(self, size, offset):
         self.offset = offset
         UIGraphicsItem.__init__(self)
         self.setAcceptedMouseButtons(QtCore.Qt.MouseButton.NoButton)
-        self.brush = QtGui.QBrush(QtGui.QColor(200,0,0))
+        self.brush = QtGui.QBrush(QtGui.QColor(255,255,255,100)) # background color
```

Comment:
self.brush was previously never used, This now holds the brush that draws the legend background.